test/partisan_SUITE.erl

```diff
@@ -38,7 +38,7 @@
 -include_lib("kernel/include/inet.hrl").
 
 -define(APP, partisan).
--define(CT_SLAVES, [rita, sue, bob, jerome]).
+-define(CLIENT_NUMBER, 3).
```

Comment:
I think we can run the symmetric checking codes after all nodes' forward_join messages finish the random walk, but I am not sure how we can check it yet.